python/py-luma.emulator/Portfile

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+name                py-luma.emulator
+github.setup        rm-hull luma.emulator 1.5.0
+revision            0
+categories          python
+maintainers         nomaintainer
+platforms           any
+supported_archs     noarch
+license             MIT
+
+description         Drawing and text rendering for small displays, emulator \
+                    module
+long_description    luma.emulator provides a series of pseudo-display devices \
+                    which allow the luma.core components to be used without \
+                    running a physical device.
+
+homepage            https://github.com/rm-hull/luma.core
```

Comment:
redundant, already set by the `github` PG, besides that it doesn't look correct anyway